xrpl/core/binarycodec/types/xchain_attestation_batch.py

```diff
@@ -0,0 +1,95 @@
+"""Codec for serializing and deserializing bridge fields."""
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional, Tuple, Type, Union
+
+from xrpl.core.binarycodec.binary_wrappers.binary_parser import BinaryParser
+from xrpl.core.binarycodec.exceptions import XRPLBinaryCodecException
+from xrpl.core.binarycodec.types.serialized_type import SerializedType
+from xrpl.core.binarycodec.types.st_array import STArray
+from xrpl.core.binarycodec.types.xchain_bridge import XChainBridge
+
+_TYPE_ORDER: List[Tuple[str, Type[SerializedType]]] = [
```

Comment:
Should this type order refer to the `definitions.json` in order to stay consistent about the order of these transactions?

Reply:
Is there any doc that'd be good to link to in order to explain why this order is the way it is?

Reply:
Nope, just the code: https://github.com/XRPLF/rippled/blob/a629214df9b2a6f3a3eedb027c54b693b12977f3/src/ripple/protocol/impl/STXChainBridge.cpp#L129-L136
The reason is that it was easier for Scott to do. I'd like that to change, so that binary visualizers that just use the definitions.json file can still process these objects/transactions, but this is the current status.